spec/mars_spec.rb

```diff
@@ -0,0 +1,12 @@
+$LOAD_PATH << File.expand_path('. ./lib', __FILE__)
+
+require 'mars'
+
+RSpec.describe Mars do
+
+  it "has to be bigger than 0x0" do
```

Comment:
I would also add a test for the positive case, where the size is correct. `it "has a size" do`